src/harness/harness.ts

```diff
@@ -1290,11 +1290,25 @@ namespace Harness {
         }
 
         export function getErrorBaseline(inputFiles: ReadonlyArray<TestFile>, diagnostics: ReadonlyArray<ts.Diagnostic>, pretty?: boolean) {
+            let outputLines = "";
+            const gen = iterateErrorBaseline(inputFiles, diagnostics, pretty);
+            for (let {done, value} = gen.next(); !done; { done, value } = gen.next()) {

```

Comment:
why can't we use for-of? Is it that we target es5, or because es6's spec doesn't support it?

Reply:
We do not compile with `--downlevelIteration` on (and we do not include a `Symbol.iterator` polyfill)